File: Week 1/Agree.c
#include <cs50.h>
#include <stio.h>

int main(void)
{
    //prompt user to agree
    char c = get_char("Do you agree? "); //single characters MUST use single quotes

    //check whether user agreed
    if (c == 'y' || c == 'Y') //|| is or, && is and
    {
        printf("Agreed.\n");
    }
    else if (c == 'n' || c == 'N') //have to ask complete question (not c == 'y' or 'Y')
    {
        printf("Not agreed.\n");
    }
}

File: Week6/agree.py
from cs50 import get_string

s = get_string("Agree?").lower()

# if s == "Y" or s == "y":
#     print("Agreed")
# if s == "N" or s == "n":
#     print("Not agreed")

# if s in ["Y", "y", "yes"]:
#     print("Agreed")
# if s in ["N", "n","no"]:
#     print("Not agreed")

if s in ["y", "yes"]:
    print("Agreed")
if s in ["n","no"]:
    print("Not agreed")